components/User-feed-status.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import Icon from './icon';
```

Comment:
Filename in the path should be capitalized (`./Icon` instead of `./icon`). That probably works on your machine if you have a case-insensitive file system, but will fail for other systems.